### PR TITLE
Adding support for project_admin and cloud_admin roles in Swift

### PR DESCRIPTION
**Pull request by ryshah**
Note project_admin/cloud_admin roles are added as part of PR #1222 

Defect 106713 in IDS

### CODE REVIEW

File: roles/swift-proxy/templates/etc/swift/proxy-server.conf [hunk 1]
```diff
@@ -50,8 +50,8 @@ use = egg:swift#name_check
 use = egg:swift#list_endpoints
 
 [filter:keystoneauth]
-operator_roles = Member,_member_,admin
-reseller_prefix =
+operator_roles = Member,_member_,admin,cloud_admin,project_admin
```
Comment (craigtracey) [inline roles/swift-proxy/templates/etc/swift/proxy-server.conf]: Please make this configurable via the defaults mechanism.

Comment (ryshah) [inline roles/swift-proxy/templates/etc/swift/proxy-server.conf]: @craigtracey: Are you recommending only the new changes (cloud/project admin) to be configurable or entire operator_roles/reseller_admin_role?

Comment (j2sol) [inline roles/swift-proxy/templates/etc/swift/proxy-server.conf]: I would template the entire value for operator_roles.

**Comment by ryshah** [comment]
@j2sol @craigtracey Added default options in proxy role

**Pull request merged**